microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -641,52 +610,88 @@ See also:
 
 
 <!-- ====================================================================== -->
-## Enable breadcrumbs in the Performance Panel
+## Enable Autofill view
 <!-- no (?) link -->
 
 _new as of Microsoft Edge 121_
 
-Adds the ability to zoom in on time ranges of a recording in the **Application** tool. Also adds a timing-breadcrumbs path to go back to previous ranges.
+This checkbox label has been changed from **Enable Autofill view** to **Autofill panel**.  See [Autofill panel](#autofill-panel), below.
+
+Status:
+* This checkbox is not present in Microsoft Edge Canary 125.
+* This checkbox is present in Microsoft Edge Stable 123.
 
-To zoom in on a time range:
 
-1. To create a time range, drag your mouse in the **Overview** section.
-1. Hover over the selected time range, and then click the magnifier icon that appears on hover. A new item is added to the timing-breadcrumbs path.
+<!-- ====================================================================== -->
+## Autofill panel
+<!-- no (?) link -->
+<!-- got moved down -->
+
+_new as of Microsoft Edge 125_
 
-To zoom out to a previous time range, click the item in the timing-breadcrumbs path.
+Adds an **Autofill** tool to DevTools, to record various events that the browser triggers when a user is filling in a form by using the browser's autofill feature.  For example, when the browser remembers your address and autofills it in an address field.
 
-![The Application tool, showing a performance recording. There is a timing breadcrumbs path at the top and a zoom icon on the selected range](./index-images/timing-breadcrumbs-path.png)
+![The Autofill tool](./index-images/autofill-tool.png)
 
 See also:
-* [Introduction to the Performance tool](../evaluate-performance/index.md)
+* [Autofill](../../privacy-whitepaper/index.md#autofill) in _Microsoft Edge Privacy Whitepaper_.
+* [Autofill](../../webview2/concepts/overview-features-apis.md?tabs=dotnetcsharp#autofill) in _Overview of WebView2 features and APIs_.
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is present in Microsoft Edge Canary 125.
+* This checkbox is present in Microsoft Edge Stable 123, as **Enable Autofill view**.
 
 
 <!-- ====================================================================== -->
-## Enable Autofill view
+## Timeline: Show postMessage dispatch and handling flows
 <!-- no (?) link -->
 
-_new as of Microsoft Edge 121_
+_new as of Microsoft Edge 124_
 
-Adds an **Autofill** tool to DevTools, to record various events that the browser triggers when a user is filling in a form by using the browser's autofill feature.  For example, when the browser remembers your address and autofills it in an address field.
+Distinguishes postMessage events from other timeline events, to easily investigate performance issues related to post messages across various threads in applications.
```

Comment:
```suggestion
Distinguishes events that are triggered by the `postMessage` method from other events displayed in the **Performance** tool, to easily investigate performance issues that are related to postin messages across various threads of an application.
```

Reply:
todo before Resolve: update What's New PR per this rewording.
```
Distinguishes events that are triggered by the `postMessage` method from other events that are displayed in the **Performance** tool, to easily investigate performance issues that are related to posting messages across various threads of an application.
```